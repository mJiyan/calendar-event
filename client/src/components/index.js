import Header from './Header';
import Month from './Month';
import Day from './Day';
import EventModal from './EventModal';

export { Header, Month, Day, EventModal };
