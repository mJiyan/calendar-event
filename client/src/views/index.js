import Calendar from './Calendar/Calendar';

export { Calendar };
