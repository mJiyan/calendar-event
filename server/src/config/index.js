const options = require('./swagger.config'),
      logger = require('./logger.config');

require('dotenv').config();

const { DB_URI, PORT, PRIVATE_KEY } = process.env;

const api = {
  port: PORT || 8081,
  jwtSecretKey: PRIVATE_KEY,
  dbUri: DB_URI,
  prefix: "/api",
  specs: "/docs",
}


module.exports = {options, logger, api};