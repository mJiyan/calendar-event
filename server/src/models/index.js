const Event = require('./eventModel');

module.exports = { Event };
