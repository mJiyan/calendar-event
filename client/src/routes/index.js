import { Calendar } from '../views';

const routes = [{ path: '/', component: Calendar }];

export default routes;
