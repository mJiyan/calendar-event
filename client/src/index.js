import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { QueryClientProvider, QueryClient } from 'react-query';
import { Provider } from 'react-redux';
import configureStore, { history } from './configureStore';
import routes from './routes';
import './assets/style/tailwind.css';

export const store = configureStore({});
const queryClient = new QueryClient();

ReactDOM.render(
  <QueryClientProvider client={queryClient}>
    <Provider store={store}>
      <Router history={history}>
        <Switch>
          {routes.map((prop) => (
            <Route exact path={prop.path} component={prop.component} key={prop.path} />
          ))}
        </Switch>
      </Router>
    </Provider>
  </QueryClientProvider>,
  document.getElementById('root'),
);
