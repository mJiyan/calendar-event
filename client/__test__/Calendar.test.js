import React from 'react';
import { Calendar } from '../src/views';
import { mount, shallow } from 'enzyme';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';

const mockStore = configureMockStore([thunk]);
const initialState = {};
const store = mockStore(initialState);

describe('rendering components', () => {
  test('it should match with the calendar component', () => {
    const component = shallow(
      <Provider store={store}>
        <Calendar />
      </Provider>,
    );
    expect(component).toMatchSnapshot();
  });
});
