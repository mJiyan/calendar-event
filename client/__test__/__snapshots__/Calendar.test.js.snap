// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering components it should have the header 1`] = `ShallowWrapper {}`;

exports[`rendering components it should match with the calendar component 1`] = `ShallowWrapper {}`;
